src/OMSimulatorLib/Types.h

```diff
@@ -98,6 +98,7 @@ typedef enum {
 } oms_element_type_enu_t;
 
 typedef enum {
+  oms_system_none,
```

Comment:
Good. I was thinking to add this.
Maybe we should also add `oms_element_none ` in `oms3_element_enu_t`.

Reply:
Probably a good idea.